frontend/router/paths.ts

```diff
@@ -58,6 +58,10 @@ export default {
   SOFTWARE_OS_DETAILS: (id: number): string => {
     return `${URL_PREFIX}/software/os/${id}`;
   },
+  SOFTWARE_VULNERABILITIES: `${URL_PREFIX}/software/vulnerabilities`,
+  SOFTWARE_VULNERABILITY_DETAILS: (cve: string): string => {
```

Comment:
this will overlap with my work, so we'll just address that when merging whichever comes in after the other

Reply:
Mine too, naming's good though?